script.js

```diff
@@ -1,59 +1,327 @@
 const nav = document.querySelector('.page-flow');
 const ham = document.querySelector('.hamburger');
 const body = document.querySelector('body');
-const li = document.querySelectorAll('.page-flow > li');
-const sec = document.querySelectorAll('*');
-let id;
+const show = document.querySelector('#showDetails');
+const btnCollapse = document.querySelector('.closeBtn');
+const head = document.querySelector('header');
 let screenWidth = window.innerWidth;
 
+const items = [];
+let instance = {
+  title: 'Tonic',
+  details: [
+    'CANOPY',
+    'Back End Dev',
+    '2015',
+  ],
+  info: 'A daily selection of privately personalized reads; no accounts or sign-ups required.',
+  tech: [
+    './images/Tag.png',
+    './images/Tag (1).png',
+    './images/Tag(2).png',
+  ],
+  src: 'img1',
+  url: [
+    './images/SnapshootPortfolio.png',
+    './desktop-images/SnapshootPortfolio.png',
+  ],
+  buttons: [
+    [
+      'https://shakir-hussain12.github.io/Mobile-Portfolio/',
+      'https://github.com/Shakir-Hussain12/Mobile-Portfolio',
+    ],
+    [
+      './desktop-images/Live_btn.png',
+      './desktop-images/Source_Btn.png',
+    ],
+  ],
+};
+items.push(instance);
+
+instance = {
+  title: 'Multi Post Stories',
+  details: [
+    'Facebook',
+    'Full Stack Dev',
+    '2015',
+  ],
```

Comment:
- [ ] Great effort implementing some of the changes requested by the previous reviewer 👋 👋, but I would suggest you implement what he suggested you do here. Kindly refactor your code to be more dynamic here and rather than having to do `items.push(instance);` of each `card`, you can easily store all details inside an `array` and `loop` through each array to show on your page when the `See Project` button is clicked

For Example 👇 

```JS
 const projects = [
  {
    title: 'Tonic',
    devs: ['Facebook', ' • backend', ' • 2021'],
    description:
      "Lorem Ipsum is simply dummied text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essent,",
    tags: ['html', 'css', 'javascript'],
    image: 'image.png',
    liveVersion: 'See Live',
    sourceLink: 'See Source',
    btnImg1: 'image.png',
    btnImg2: 'images.png',
  },
  {
    title: 'Tonic',
    devs: ['Facebook', ' • backend', ' • 2021'],
    description:
      "Lorem Ipsum is simply dummied text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essent,",
    tags: ['html', 'css', 'javascript'],
    image: 'image.png',
    liveVersion: 'See Live',
    sourceLink: 'See Source',
    btnImg1: 'image.png',
    btnImg2: 'images.png',
  },
  {
    title: 'Tonic',
    devs: ['Facebook', ' • backend', ' • 2021'],
    description:
      "Lorem Ipsum is simply dummied text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essent,",
    tags: ['html', 'css', 'javascript'],
    image: 'image.png',
    liveVersion: 'See Live',
    sourceLink: 'See Source',
    btnImg1: 'image.png',
    btnImg2: 'images.png',
  },
,
...
```